drivers/marvell/uart/a3700_console.S

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (C) 2016 Marvell International Ltd.
+ *
+ * SPDX-License-Identifier:	BSD-3-Clause
+ * https://spdx.org/licenses
+ */
+
+#include <asm_macros.S>
+#include <a3700_console.h>
+
+	.globl	console_core_init
```

Comment:
The marvel platforms will need to use MULTI_CONSOLE_API. Can be done as a later PR.

Reply:
Yes, I know and agree with you. Need to get a time assignment to do this job